### PR TITLE
Correct Grammar in CoreKill ReadMe & Explain HM3InGameTools

### PR DESCRIPTION
**Pull request by HHCHunter**
Fixed some grammar in the CoreKill ReadMe.md
Explain how HM3InGameTools works

### CODE REVIEW

**Comment by DronCode** [comment]
The `m_realNosieLevel` is a temporary name (ZOSD component not reversed complexly). It used in HM3CoreKill/HM3CoreKill/ck/HM3InGameTools.cpp

**Comment by HHCHunter** [comment]
> 
> 
> The `m_realNosieLevel` is a temporary name (ZOSD component not reversed complexly). It used in HM3CoreKill/HM3CoreKill/ck/HM3InGameTools.cpp

Ah, I see.

Finishing off ZOSD would be best here.

File: HM3CoreKill/HM3CoreKill/ck/HM3InGameTools.cpp [hunk 1]
```diff
@@ -129,7 +129,7 @@ namespace ck
 				ImGui::Text("Noise level: "); ImGui::SameLine(0.f, 15.f);
 
 				ImVec4 noiseLevelColor = ImVec4(0.f, 1.f, 0.f, 1.f);
-				if (osd->m_realNosieLevel >= 0.f && osd->m_realNosieLevel <= 40.f)
```
Comment (DronCode) [inline HM3CoreKill/HM3CoreKill/ck/HM3InGameTools.cpp:132]: Left part of condition (`osd->m_realNosieLevel >= 0.f`) should be changed too.
Also as other cases further.

**Review by DronCode — APPROVED** [review]

**Comment by HHCHunter** [comment]
Hopefully my explanation of the collapse header is correct?

**Comment by DronCode** [comment]
Looks nice, I can fix merge conflict manually. 
Leave a comment when you will be ready to pull your changes into the master branch.

**Comment by HHCHunter** [comment]
Ready for merge, apologies for the conflicts

**Comment by DronCode** [comment]
Yes, conflict fixed.

**Pull request merged**